.devcontainer/setup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Install Python and pip
+sudo apt-get install software-properties-common
+sudo add-apt-repository ppa:deadsnakes/ppa
+sudo apt-get update
+sudo apt-get install python3.10 python3-pip
+
+# Install poetry
+curl -sSL https://install.python-poetry.org | python3 -
+poetry config virtualenvs.in-project true
+
+# Initialize submodules
+git submodule update --init
+
+# Don't ask for superuser access when debugging extensions
+# See https://github.com/benibenj/vscode-pythonCpp/issues/18
+echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope
```

Comment:
Without this command I couldn't get the debugger to work (see related issue). I honestly don't know what this does and if this is a security issue. It would be nice to get the debugger to work on Codespaces but it's not a must-have for me.

Reply:
Never mind. The change in [5e37607](https://github.com/PyVRP/PyVRP/pull/580/commits/5e37607d4c208f5bf12f1017b99f467672a130cf) works.